app/scripts/modules/core/src/pipeline/config/triggers/pubsub/pubsub.trigger.ts

```diff
@@ -1,5 +1,7 @@
 import { IController, module } from 'angular';
 
+import { SETTINGS } from '@spinnaker/core';
```

Comment:
because this file is inside `core` itself, this should be imported from `'core/config/settings'`.